Frontend/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import Header from "./Header"
+import Main from "./Main"
+import LoginForm from "./LoginForm"
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link
+  } from "react-router-dom";
+
+// import Footer from "./Footer"
+class App extends React.Component {
+    // constructor() {
```

Comment:
Try to clean up your code. Also, see the warnings in the console, they will show you what you should delete or not ( useless imports )